### DISCUSSION TITLE
Lazy loading of images

### DISCUSSION THREAD

**Discussion by JeremieLitzler** [vuepress-theme-hope/vuepress-theme-hope#2341] [Q&A] [answered]
Hello,

I see a lot of great features in the theme.

But what about lazy loading of images?

Thanks :)

**Comment by Mister-Hope** [comment, marked as answer]
https://vuepress-theme-hope.github.io/v2/config/plugins/md-enhance.html#lazyload

**Reply by JeremieLitzler** [reply]
Oh no, sorry for not searching better. I searched "lazy" keyword and I though I didn't anything... Thanks @Mister-Hope :)

**Reply by Mister-Hope** [reply]
This feature is provided since v1, I just renamed this to `imageLazyload` and add missing documatation in "Guide part", it will be a BREADKING CHANGE in next version to align `imageTitle` `imageSize` and `imageMark`.
